### PR TITLE
[#14120] Remove Ropsten and Rinkeby test networks

### PR DESCRIPTION
**Pull request by rasom**
fix partially #14120

status: ready

### CODE REVIEW

**Comment by Parveshdhull** [comment]
> No more code on both status-go and status-mobile side, associated with these networks

Hi @rasom, do we also need to make changes in status-go?

**Comment by rasom** [comment]
@Parveshdhull yep, likely we do, missed that point

**Comment by flexsurfer** [comment]
is it wip ? or should we review it ?

**Comment by rasom** [comment]
@churik 
> https://github.com/status-im/status-mobile/blob/61cb54fb8e1358fe4a13a3d8330817436e29bbc0/test/appium/pytest.ini#L14

should Ropsten be replaced with Goerli in this line?

**Comment by rasom** [comment]
@flexsurfer wip

**Comment by rasom** [comment]
@churik @pavloburykh  @qoqobolo 
I can try to clean up those files on my own but it probably would be better if you do that https://github.com/status-im/status-mobile/search?l=Python&q=ropsten

**Comment by qoqobolo** [comment]
> transaction: all cases that are related to transaction completion in Ropsten network 
should Ropsten be replaced with Goerli in this line?

> I can try to clean up those files on my own but it probably would be better if you do that https://github.com/status-im/status-mobile/search?l=Python&q=ropsten

Hey @rasom, please skip all these test files for now, we'll clean up them separately. 
So you can merge this PR as it is once it's ready (ping us for manual QA if needed).
Thank you

**Review by J-Son89 — APPROVED** [review]

File: src/status_im/utils/config.cljs [hunk 1]
```diff
@@ -21,13 +21,8 @@
...
 
-(def mainnet-rpc-url
-  (str "https://eth-archival.gateway.pokt.network/v1/lb/" POKT_TOKEN))
-
-(def testnet-rpc-url
-  (str "https://ropsten.infura.io/v3/" POKT_TOKEN))
-(def goerli-rpc-url
-  (str "https://goerli-archival.gateway.pokt.network/v1/lb/" POKT_TOKEN))
+(def mainnet-rpc-url (str "https://mainnet.infura.io/v3/" POKT_TOKEN))
+(def goerli-rpc-url  (str "https://goerli.infura.io/v3/" POKT_TOKEN))
```
Comment (flexsurfer) [inline src/status_im/utils/config.cljs]: is it intentional? pokt url replaced by infura

Comment (rasom) [reply src/status_im/utils/config.cljs]: nope it is not, thanks

Comment (rasom) [reply src/status_im/utils/config.cljs]: fixed

**Review by flexsurfer — APPROVED** [review]

**Pull request merged**